package.json

```diff
@@ -33,29 +33,30 @@
     "babel-preset-stage-0": "6.16.0",
     "babel-register": "6.18.0",
     "babel-runtime": "6.18.0",
-    "enzyme": "2.5.1",
+    "enzyme": "3.1.0",
+    "enzyme-adapter-react-16": "^1.0.1",
     "eslint": "3.8.1",
     "eslint-plugin-react": "6.4.1",
     "flow-bin": "^0.33.0",
     "jest": "^18.1.0",
-    "react": "15.4.0",
+    "react": "^16.0.0",
     "react-addons-test-utils": "^15.3.2",
-    "react-dom": "15.4.0",
+    "react-dom": "16.0.0",
     "react-hot-loader": "^1.3.1",
+    "react-test-renderer": "^16.0.0",
     "rimraf": "^2.5.4"
   },
   "dependencies": {
     "ldclient-js": "^1.1.8",
-    "react-broadcast": "^0.3.1"
+    "react-broadcast": "^0.5.2"
   },
   "peerDependencies": {
-    "react": ">=15.0.0",
-    "react-dom": ">=15.0.0"
+    "react": ">=16.0.0",
+    "react-dom": ">=16.0.0"
```

Comment:
I'm still not 100% on how NPM resolve dependencies but does this mean that `react-launch-darkly` would only be supporting apps that are using react 16?